src/ks/hangedman/Game.java

```diff
@@ -24,24 +30,40 @@ public void start() {
             if (!decision.equalsIgnoreCase("y")) {
                 break;
             }
-
         }
-
     }
 
     private List<String> readFile(String fileName) {
         List<String> wordsFromFile = new ArrayList<>();
         try (Scanner reader = new Scanner(Paths.get(fileName))) {
             while (reader.hasNextLine()) {
-                String row = reader.nextLine();
-                wordsFromFile.add(row);
+                String row = reader.nextLine().toLowerCase();
+                String[] singleWords = row.split("\\s+");
+                if (singleWords.length > 1) {
```

Comment:
this could be simplified:

if (length == 1) or else

because in both conditions the code is the same now (when <1 or >1)